### PR TITLE
OboeTester: Add extra newlines in device report for legibility

### PR DESCRIPTION
**Pull request by robertwu1**
Before change:
```
Device Report:
OboeTester (59) v 2.3.0, Oboe v 1.6.2Google, Pixel 4a (5G), bramble
==== Device ==================== 2
Id : 2
```

After change:
```
Device Report:
App: OboeTester (59) v 2.3.0, Oboe v 1.6.2
Phone: Google, Pixel 4a (5G), bramble

==== Device ==================== 2
Id : 2
```

### CODE REVIEW

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DeviceReportActivity.java [hunk 1]
```diff
@@ -103,8 +103,8 @@ private void reportDeviceInfo(Collection<AudioDeviceInfo> devices) {
         logClear();
         StringBuffer report = new StringBuffer();
         report.append("Device Report:\n");
-        report.append(MainActivity.getVersiontext());
-        report.append(Build.MANUFACTURER + ", " + Build.MODEL + ", " + Build.PRODUCT);
+        report.append(MainActivity.getVersiontext()).append("\n");
```
Comment (philburk) [inline apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DeviceReportActivity.java:106]: Try to keep "tag:value" format. Maybe

    Device Report
    Oboe: Oboetester (58).....
    Build: Google, Pixel ...

**Review by philburk — CHANGES_REQUESTED** [review]

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DeviceReportActivity.java (hunk 1 above)
Comment (robertwu1) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DeviceReportActivity.java]: Changed to the following

```
App: OboeTester (59) v 2.3.0, Oboe v 1.6.2
Phone: Google, Pixel 4a (5G), bramble
```

Comment (philburk) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DeviceReportActivity.java]: I like "App:".
In our group we tend to focus on phones. But we need OboeTester to support non-phone devices such as TV and Auto. 
Maybe "Device:".

Comment (robertwu1) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DeviceReportActivity.java]: Done

**Review by philburk — APPROVED** [review]

**Pull request merged**